app/views/layouts/application.html.haml

```diff
@@ -7,4 +7,7 @@
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   %body
+    %p.notice= notice
+    %p.alert= alert
```

Comment:
What does these 2 lines do for you?

Reply:
these are to show the messages like "successfully logged in" etc...
they come with devise

Reply:
I’ll let it pass, but I would prefer a conditional here.